### DISCUSSION TITLE
Force-static support or explicit error

### DISCUSSION THREAD

**Discussion by zifeo** [amannn/next-intl#577] [Q&A]
### Description

Using `useTranslations` inside a page with `export const dynamic = 'force-static';` lead to this error:
```
Unable to find `next-intl` locale because the middleware didn't run on this request. See https://next-intl-docs.vercel.app/docs/routing/middleware#unable-to-find-locale
```

Is `force-static` planned to be supported? Otherwise, the error helper should include this case.

### Mandatory reproduction URL (CodeSandbox or GitHub repository)

https://codesandbox.io/p/sandbox/next-intl-bug-template-app-forked-xhs3mk?file=%2Fsrc%2Fapp%2F%5Blocale%5D%2Fpage.tsx%3A9%2C1

### Reproduction description

1. Open reproduction

### Expected behaviour

No error

**Comment by amannn** [comment]
I think you're looking for this: https://next-intl-docs-git-feat-next-13-rsc-next-intl.vercel.app/docs/getting-started/app-router#static-rendering

I'll move this to a discussion, since this appears to be a usage question.

**Reply by zifeo** [reply]
@amannn I believe the errors should mention something about static rendering, and I am not clear whether `unstable_setRequestLocale` should also apply to `force-static` (with other pages are not static).
